### DISCUSSION TITLE
Is there a way to Orphan tenants before uninstalling flux?

### DISCUSSION THREAD

**Discussion by walter-staked** [fluxcd/flux2#3057] [Q&A]
Afternoon all,
Just curious if there's a way to orphan tenant deployments before uninstalling flux.  with 
```shell 
flux uninstall 
```

**Comment by kingdonb** [comment]
Greetings,

`flux uninstall` should orphan all the tenants (they will not be managed by Flux anymore, but they will be left behind and should still run)

If this is not exactly what you meant, please clarify the question further – the Pruning or Garbage Collection **is** pre-empted by uninstall, and does not occur when `flux uninstall` is used. Garbage collection is only performed when a Flux kustomization resource is deleted while Flux controllers are running/healthy and the `spec.prune` is set to `true`. The uninstall routine has been implemented with a specific order to avoid triggering that behavior.
